examples/DataApi-NFT/template/modules/shared_data/src/main/scala/com/my/nft_example/shared_data/Combiners.scala

```diff
@@ -0,0 +1,57 @@
+package com.my.nft_example.shared_data
+
+import com.my.nft_example.shared_data.Data.{Collection, MintCollection, MintNFT, NFT, State, TransferCollection, TransferNFT}
+import com.my.nft_example.shared_data.Utils.customUpdateSerialization
+import monocle.Monocle.toAppliedFocusOps
+import org.tessellation.schema.address.Address
+import org.tessellation.security.hash.Hash
+
+import java.util.Calendar
+import scala.collection.mutable
+
+object Combiners {
+  def combineMintCollection(update: MintCollection, acc: State, collectionOwner: Address): State = {
+    val collectionId = Hash.fromBytes(customUpdateSerialization(update)).toString
+    val nowInTime = Calendar.getInstance().getTime.getTime
```

Comment:
`System.currentTimeMillis()` is generally more preferred if not manipulating or rendering actual dates.